crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -1847,6 +1847,7 @@
                 "max_verifier_meter_ticks_per_function": {
                   "u64": "6000000"
                 },
+                "min_move_binary_format_version": null,
```

Comment:
I'm confused why this is `null`

Reply:
This is the case for protocol version fields that are null/not used at previous versions (i.e., fields that have been added over time) is my understanding.